job-descriptions/software-engineer.md

```diff
@@ -4,57 +4,46 @@
 
 ### About us
 
-Sourcegraph's mission is to enable every software developer to create products using the best technology. The innovations of the future will all rely on software, and we want to bring the future sooner by building the best tools that help developers answer questions about their code. You can learn more about our mission by reading [our master plan](https://sourcegraph.com/plan).
+We're building the new standard developer platform. Top tech companies have invested \$100Ms to build internal developer platforms for code search, code review, alerts, and automation. Sourcegraph provides this standard developer platform to every company, helping startups and large enterprises ship better software faster.

```

Comment:
I think reducing the number of times "developer platform" is written can make it better.

```suggestion
Top tech companies like Google and Facebook have invested \$100Ms to build internal developer platforms for code search, code review, alerts, and automation. Sourcegraph is bringing these capabilities to every company in the world, helping both startups and large enterprises ship better software faster.
```

Reply:
Chiming in here to add my 2 cents: I do like the previous opening sentence because it's bold, to the point and sets the tone.

Reply:
To be clear: I don't feel strong about this. It just sounded a bit alliterative.

Reply:
"$100Ms" feels somewhat strange to me, it's like using casual words in a formal writing. But keep in mind I'm not a native speaker.

Reply:
What happened to our "master plan" and "bring the future sooner"? Is it intentional we're removing these words from this "About us" page?

Reply:
> I do like the previous opening sentence because it's bold, to the point and sets the tone.

Yeah, I think this is a good property.

1. Bold open ended statement about the product we are building: "new standard developer platform". No qualifications or implied limitations. Full stop.
2. Evidence that this product is valuable (top tech companies have standard developer platforms) and a problem statement (existing dev platforms are internal and not accessible).
3. Our solution solves the problem from (2) by being open and for everyone.

One point that isn't  explicit right now is that we aren't _merely_ bringing what these companies have to everyone else, we are actually improving on it. I think the presence of (1) helps mitigate this, but that it is completely lost in your proposed changes @tsenart .

Overall, this is the language we use on our homepage so it has been thought about by Quinn and I am hesitant to change it here.

> "$100Ms" feels somewhat strange to me, it's like using casual words in a formal writing, because I often see "hundreds of millions".

Yeah it is a bit informal but I think that is ok. I also think a big number like this is more noticeable so it provides desirable emphasis (instead of blending in with the rest of the text).

> What happened to our "master plan" and "bring the future sooner"? Is it intentional we're removing these words from this "About us" page?

I linked to the master plan in the next paragraph and will think about if there is a way to specifically inline the words "bring the future sooner".